clang/test/dpct/bfloat16_ext.cu

```diff
@@ -0,0 +1,15 @@
+// UNSUPPORTED: cuda-8.0, cuda-9.0, cuda-9.1, cuda-9.2, cuda-10.0, cuda-10.1, cuda-10.2
+// UNSUPPORTED: v8.0, v9.0, v9.1, v9.2, v10.0, v10.1, v10.2
+// RUN: dpct --format-range=none --use-dpcpp-extensions=intel_device_math -out-root %T/bfloat16_ext %s --cuda-include-path="%cuda-path/include" -- -x cuda --cuda-host-only
+// RUN: FileCheck %s --match-full-lines --input-file %T/bfloat16_ext/bfloat16_ext.dp.cpp
+
+#include "cuda_bf16.h"
+
+__global__ void kernelFuncBfloat162Arithmetic() {
+  // CHECK: sycl::marray<sycl::ext::oneapi::bfloat16, 2> bf162, bf162_1, bf162_2;
+  __nv_bfloat162 bf162, bf162_1, bf162_2;
+  // CHECK: bf162 = bf162_1 / bf162_2;
```

Comment:
L11 is same as bfloat16.cu 's L34,  so the intel_device_math doesn't make difference?

Reply:
yes, now sycl math ext API cannot support bf16 type, so I use emulation too. I add this case because I need make sure when we use intel_device_math to migrate the case, it will not seg fault.